packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -489,50 +489,46 @@ class _AdaptiveScaffoldState extends State<AdaptiveScaffold> {
           bodyOrientation: widget.bodyOrientation,
           bodyRatio: widget.bodyRatio,
           internalAnimations: widget.internalAnimations,
-          primaryNavigation: widget.destinations != null &&
```

Comment:
`widget.destinations` could never be null -> `widget.destinations != null` is always true.
`widget.selectedIndex` could never be null -> `widget.selectedIndex != null` is always true.

`true && true == true`